resources/js/components/ui/Email.tsx

```diff
@@ -1,17 +1,21 @@
-import {Show, VoidComponent} from "solid-js";
+import {Show, VoidComponent, splitProps} from "solid-js";
+import {htmlAttributes} from "../utils";
 import {CopyToClipboard} from "./CopyToClipboard";
 import {EMPTY_VALUE_SYMBOL} from "./symbols";
 
-interface Props {
+interface Props extends htmlAttributes.div {
   email: string | undefined;
 }
 
 /** A component for displaying a copiable email address. No mailto. */
-export const Email: VoidComponent<Props> = (props) => (
-  <Show when={props.email} fallback={EMPTY_VALUE_SYMBOL}>
-    <div class="flex gap-1">
-      {props.email}
-      <CopyToClipboard text={props.email} />
+export const Email: VoidComponent<Props> = (props) => {
+  const [_localProps, divProps] = splitProps(props, ["email"]);
```

Comment:
No to może to allProps tutaj:

(allProps) => {
  const [props, divProps] = splitProps(allProps, ["email"]);

Reply:
Jak omówiliśmy, na razie zostaje, w pewnym momencie można by wybrać jak jest najlepiej i tak robić, ewentualnie poprawić wszędzie.